### PR TITLE
Update homepage buttons to have final livestream

### PR DESCRIPTION
**Pull request by MarkBennett**
Button text is a little big but I think it's ok for now.

Screen Shot 2020-05-14 at 10 20 32 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,12 +60,7 @@ <h1>CODEVID-19</h1>
             href="https://forms.gle/yHayrwJ2yhGjjTZD6"
             target="_blank"
             class="button primary icon solid fa-external-link-alt"
-            >Vote For a Finalist</a
-          >
-        </li>
-        <li>
-          <a href="{{ '/join-the-fight' | relative_url }}" class="button"
-            >Join the Fight</a
+            >Watch the Final Award Presentation</a
```
Comment (ajyong) [inline index.html:63]: Maybe "Watch the Final Awards" is enough

Comment (ajyong) [reply index.html:63]: I can't add a comment on L57 for some reason, but I think I prefer how it looks when `fit` is removed as a class.  I only used `fit` when we had 3-4 buttons.

**Review by ajyong — CHANGES_REQUESTED** [review]

**Review by ajyong — APPROVED** [review]
But really this is good to go.  I'm nitpicking and fine with the squash and merge.

File: index.html (hunk 1 above)
Comment (MarkBennett) [reply index.html:63]: Ok. Why don't I merge and you can tweak the classes as I'm not familiar with what `fit` does.

**Pull request merged**